packages/cms-lib/oauth.js

```diff
@@ -31,6 +32,34 @@ const getOauthManager = (portalId, portalConfig) => {
   return oauthManagers.get(portalId);
 };
 
+const setupOauth = (portalId, portalConfig) => {
+  const config = getPortalConfig(portalId) || {};
+  return new OAuth2Manager(
+    {
+      ...portalConfig,
+      environment: config.env || 'prod',
+    },
+    logger
+  );
+};
+
+const addOauthToPortalConfig = (portalId, oauth) => {
+  logger.log('Updating configuration');
+  try {
+    updatePortalConfig({
+      ...oauth.toObj(),
+      authType: AUTH_METHODS.oauth.value,
+      portalId,
+    });
+    logger.log('Configuration updated');
+    process.exit();
```

Comment:
This looks unexpected. Could we move this as well as the error handling to where this is called?